canonical-web-and-design/jaas.ai
# Upgrade dependencies.

## Done

Upgrade dependencies to resolve some of the vulnerable package warnings. None of these vulnerable packages are exposed to the client.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Does it work as it should?

## Details

Fixes #402
